code/modules/clothing/suits/cloaks.dm

```diff
@@ -18,43 +18,101 @@
 	flags_1 = NODROP_1
 	flags_inv = HIDEHAIR|HIDEEARS
 
-/obj/item/clothing/neck/cloak/suicide_act(mob/user)
+/obj/item/clothing/suit/hooded/cloak/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is strangling [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return(OXYLOSS)
 
-/obj/item/clothing/neck/cloak/hos
+/obj/item/clothing/suit/hooded/cloak/hos
 	name = "head of security's cloak"
 	desc = "Worn by Securistan, ruling the station with an iron fist."
 	icon_state = "hoscloak"
+	hoodtype = /obj/item/clothing/head/hooded/cloakhood/hos
+	allowed = list(/obj/item/melee/baton,/obj/item/melee/classic_baton,/obj/item/grenade,/obj/item/gun,/obj/item/reagent_containers/spray/pepper,/obj/item/restraints/handcuffs,/obj/item/device/assembly/flash/handheld,/obj/item/clothing/glasses,/obj/item/ammo_casing/shotgun,/obj/item/ammo_box,/obj/item/reagent_containers/food/snacks/donut,/obj/item/reagent_containers/food/snacks/donut/jelly,/obj/item/kitchen/knife/combat,/obj/item/device/flashlight/seclite,/obj/item/melee/classic_baton/telescopic,/obj/item/device/radio,/obj/item/clothing/gloves/,/obj/item/restraints/legcuffs/bola)
 
-/obj/item/clothing/neck/cloak/qm
+/obj/item/clothing/head/hooded/cloakhood/hos
+	icon_state = "hoshood"
+	armor = list(melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0, fire = 70, acid = 90)
```

Comment:
this is really absurd

Reply:
yeah that fire and acid resistance is too high